flow-server/src/main/java/com/vaadin/flow/server/connect/VaadinConnectController.java

```diff
@@ -153,6 +161,17 @@ public VaadinConnectController(
         }
     }
 
+    private ObjectMapper getDefaultObjectMapper(ApplicationContext context) {
```

Comment:
```suggestion
    private ObjectMapper createVaadinConnectObjectMapper(ApplicationContext context) {
```
* I would rename the get to create, because we create a new instance
* I would rename the method to show that we create the default ObjectMapper for VaadinConnect, not the default ObjectMapper